internal/controllers/clusterextension_controller.go

```diff
@@ -84,7 +84,7 @@ type Applier interface {
 }
 
 type InstalledBundleGetter interface {
-	GetInstalledBundle(ctx context.Context, ext *ocv1alpha1.ClusterExtension) (*ocv1alpha1.BundleMetadata, error)
+	GetInstalledBundle(ctx context.Context, ext *ocv1alpha1.ClusterExtension) (*ocv1alpha1.BundleMetadata, release.Status, error)
```

Comment:
I'd say only add this if we actually need it. Looks like we aren't using it yet.

And I don't think we need to. IIRC, yesterday we said this function would return the most recent revision that has status==Deployed, otherwise return `nil` (or an error).